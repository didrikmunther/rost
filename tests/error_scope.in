let c = 2

if(c < 5) {
	let a = 5
}

if(c < 5) {
	a = 2
}