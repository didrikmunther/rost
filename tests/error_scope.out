  --> ./tests/error_scope.in:8:2 => CompilerError
7 | if(c < 5) {
8 |     a = 2
  |     ^
  |     └─ Undefined variable: a
9 | }